CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Properties `num_time_steps_adjoint` and `tmesh_adjoint` to `JaxSimulation` to estimate adjoint run time.
 - Ability to add `path` to `updated_copy()` method to recursively update sub-components of a tidy3d model. For example `sim2 = sim.updated_copy(size=new_size, path="structures/0/geometry")` creates a recursively updated copy of `sim` where `sim.structures[0].geometry` is updated with `size=new_size`.
 - Python 3.12 support. Python 3.8 deprecation. Updated dependencies.
+- A batch of ModeSolver objects can be run concurrently using `tidy3d.plugins.mode.web.run_batch()`
```

Comment:
Somehow the changes to Changelog.md was not implemented. I think you wanted this sentence to be
A batch of `ModeSolver` objects can be run concurrently using `tidy3d.plugins.mode.web.run_batch()`.